setup.py

```diff
@@ -23,7 +23,7 @@
         ("share/openfisca/openfisca-country-template", ["CHANGELOG.md", "LICENSE", "README.md"]),
         ],
     install_requires = [
-        "OpenFisca-Core[web-api] >= 27.0, < 28.0",
+        "OpenFisca-Core[web-api] >=27.0,<32.0",
```

Comment:
@Morendil @fpagnoux When the `build` step is successful on multiple versions, do we agree on keeping a large interval of compatibility with core or would you advise against it (aka upgrading Core minor version to 31 here)? 🤔

Reply:
@sandcha I can't think of very strong reasons to do one or the other. It seems good to keep in sync with the most recent version of Core, and at the same time it's useful to know that there are several recent versions which did not break the country template. And if we let Dependabot do its thing we have less work to do.

Reply:
In some repositories, I see a risk of accepting a dependency that gets a fixup in its upgraded version. This might not apply to the country template though as, for example, it would require to install openfisca-core manually (like v30) and, later, install the country template manually (that didn't pass its own tests on v30). 🤔